nip47/event_handler_test.go

```diff
@@ -39,7 +39,7 @@ func TestCreateResponse(t *testing.T) {
 
 	reqEvent.ID = "12345"
 
-	ss, err := nip04.ComputeSharedSecret(reqPubkey, svc.Keys.GetNostrSecretKey())
+	nip47Cipher, err := cipher.NewNip47Cipher("1.0", reqPubkey, svc.Keys.GetNostrSecretKey())
```

Comment:
Can we add another test method for `TestCreateResponse_Nip04` similar to this one?

Reply:
True, will add it!